taxonomies/additives.txt

```diff
@@ -3354,12 +3354,12 @@ wikipedia:en:https://en.wikipedia.org/wiki/Calcium_carbonate
 # ingredient/calcium-carbonate has 4220 products in 10 languages @2019-02-03
 # mineral/calcium-carbonate has 4301 products @2019-02-03
 e_number:en:170
-mandatory_additive_class:en: en:acidity-regulator, en:anti-caking-agent, en:stabilizer, en:firming-agent, en:flour-treatment-agent, en:glazing-agent, en:colour
```

Comment:
It looks like you manually changed  all occurences of en:stabilizer to en:stabiliser, but did not run "make lint_taxonomies" fully on additives.txt ?

Reply:
Because I wanted to make it in another PR (but needed this stabliser change to have test pass !